src/system.py

```diff
@@ -1007,6 +1007,29 @@ def getServices(system):
                     }
                 }
             },
+            'GET_SOLAR_SWITCH_STATUS': {
+                'about': "Gets the status of the switch connected to the power of the solar panel currentsense ICs",
```

Comment:
How come these "about" fields didn't get reflected in commanddocs?

Reply:
Wrong key value. Fixed it